src/test/xmeta/helpers.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "antlr4-runtime.h"
+#include "XlangParser.h"
+#include "XlangLexer.h"
+
+using namespace antlr4;
```

Comment:
Don't drag entire namespaces into global scope in a header.
http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rs-using-directive